test/utils/describeConformanceV5.js

```diff
@@ -251,14 +280,40 @@ const fullSuite = {
 export default function describeConformanceV5(minimalElement, getOptions) {
   const { after: runAfterHook = () => {}, only = Object.keys(fullSuite), skip = [] } = getOptions();
 
+  const filteredTests = Object.keys(fullSuite).filter(
+    (testKey) => only.indexOf(testKey) !== -1 && skip.indexOf(testKey) === -1,
+  );
+
+  it('should throw error if run the tests without required property', () => {
+    expect(() => validateOptions({}, ['themeVariants'])).to.throw();
+    expect(() => validateOptions({}, ['themeStyleOverrides'])).to.throw();
+    expect(() => validateOptions({}, ['themeDefaultProps'])).to.throw();
+  });
+
+  it('should not throw error if provided required properties (does not care the type)', () => {
+    expect(() =>
+      validateOptions({ render: () => {}, muiName: 'Mui' }, [
+        'themeVariants',
+        'themeStyleOverrides',
+        'themeDefaultProps',
+      ]),
+    ).not.to.throw();
+  });
+
+  it('should not throw error if the tests does not need required property (does not care the type)', () => {
+    expect(() => validateOptions({}, ['propsSpread'])).not.to.throw();
+  });
```

Comment:
This is not flying. We would run the same tests x the number of components, so about x100

Reply:
Yeah, I missed that point. moved to `describeConformanceV5.test.js`.